lstchain/calib/camera/r0.py

```diff
@@ -412,6 +450,33 @@ def _load_calib(self):
                 self.pedestal_value_array[:, :, self.size4drs:self.size4drs + 40] \
                     = pedestal_data[:, :, 0:40] - self.offset
 
+        if not self.calibration_path:
+            self.log.info("no dc to pe conversion done as no calibration file as been given")
+            return
+
+        self.mon_data.tels_with_data = self.allowed_tels
+        self.log.info(f"read {self.calibration_path}")
+        try:
+            with HDF5TableReader(self.calibration_path) as h5_table:
+                assert h5_table._h5file.isopen == True
```

Comment:
I don't think you need that assert here. The first line should raise an exception if opening the file does not work. Also, iy might be good to specify `mode=r`